src/components/Accounts/Menu.js

```diff
@@ -15,7 +15,7 @@ import {
 import { refundClaimReturned } from '../../constants';
 
 import css from './Menu.css';
-import { useLocalizeCurrency } from '../../hooks/useLocalizedCurrency/useLocalizeCurrency';
+import useLocalizeCurrency from '../../hooks/useLocalizedCurrency/useLocalizeCurrency';
```

Comment:
after updating `src/hooks/index.js` file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import